jest.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     '<rootDir>[/\\\\](|coverage|node_modules)[/\\\\]',
     'src/components/TrackTile/TrackerDetails/AdvancedTrackerEditor/AdvancedTrackerEditor.test.tsx',
   ],
-  preset: 'react-native',
+  preset: 'jest-expo',
```

Comment:
changing this config mostly for experimentation but ideally it would be seperate and only "expo" things would need to use a different config